src/task_types/fuzzer_tasks.rs

```diff
@@ -279,6 +289,7 @@ impl GenFuzzerService for GenFuzzerServiceImpl {
 /// * `params` - Parameters for how task should be generated.
 /// * `generated_suite_name` - Name of suite to execute against.
 /// * `old_version` - Previous version of mongo to test against.
+/// * `bazel_target` - Bazel target name, if running via bazel
```

Comment:
To be more precise, is this the "short" name (eg. my_special_tests) or full name (eg. //foo/bar:my_special_tests)?

Oh - I see it's the latter in your unittest. Good to be explicit with the contract then.